src/main.tf

```diff
@@ -41,4 +41,11 @@ resource "azurerm_healthcare_fhir_service" "main" {
       image_name   = oci_artifact.value.image_name
     }
   }
+
+  cors {
```

Comment:
This feels _almost_ like no `cors`. Do we need this block for something particular, like enabling some networking thing? CORS is usually used to reduce vulnerabilities by whitelisting origins, and methods. It feels like here we're allowing all so are we cors-ing 🤷